source/NVDAObjects/window/edit.py

```diff
@@ -379,38 +368,6 @@ def _getWordOffsets(self,offset):
 				start=end
 				end=watchdog.cancellableSendMessage(self.obj.windowHandle,EM_FINDWORDBREAK,WB_MOVEWORDRIGHT,offset)
 			return (start,end)
-		elif winVersion.winVersion.major<6: #Implementation of standard edit field wordbreak behaviour (only breaks on space)

```

Comment:
Why is all this code removed?

Reply:
Because it is code that would only apply to Windows XP.